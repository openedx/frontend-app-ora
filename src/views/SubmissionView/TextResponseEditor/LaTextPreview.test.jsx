import { shallow } from '@edx/react-unit-test-utils';

import LatexPreview from './LaTexPreview';

describe('<LatexPreview />', () => {
  it('renders', () => {
    const wrapper = shallow(<LatexPreview latexValue="some latext value" />);
    expect(wrapper.snapshot).toMatchSnapshot();
  });
});
