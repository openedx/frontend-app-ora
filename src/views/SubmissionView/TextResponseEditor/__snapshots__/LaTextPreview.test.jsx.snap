// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LatexPreview /> renders 1`] = `
<div
  className="mt-2"
>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "some latext value",
      }
    }
  />
</div>
`;
